### DISCUSSION TITLE
Is it possible to show only last modified date?

### DISCUSSION THREAD

**Discussion by istvnurbn** [adityatelange/hugo-PaperMod#1131] [Q&A]
Hi,

If I set up in a page's frontmatter to hide metadata it will hide all data (like reading time, author etc.). However, I would like to show the last modified date - and that only without the message and author - which is tied to editPost parameter in the main config.

Is there a way to do that?

**Comment by RoneoOrg** [comment]
A partial answer is available there:
https://github.com/adityatelange/hugo-PaperMod/issues/1037
